services/mapper.go

```diff
@@ -31,7 +31,79 @@ const (
 	zeroAddress           = "0x0000000000000000000000000000000000000000000000000000000000000000"
 )
 
-// FeeOps returns the fee operations for a given transaction
+func parseTransferOps(startIndex int, transfers []*evmClient.EVMTransfer) []*RosettaTypes.Operation {
+	count := startIndex
+	var ops []*RosettaTypes.Operation
+	for i, transfer := range transfers {
+		if i != 0 {
+			count++
+		}
+		var address string
+		amount := transfer.Value
+		shouldAdd := true
+		if transfer.From == nil {
+			address = transfer.To.String()
+			shouldAdd = false
+		} else if transfer.To == nil {
+			address = transfer.From.String()
+			amount = new(big.Int).Neg(transfer.Value)
+			shouldAdd = false
+		}
+
+		if shouldAdd {
+			address = transfer.From.String()
+			amount = new(big.Int).Neg(transfer.Value)
+		}
+
+		singleOp := &RosettaTypes.Operation{
+			OperationIdentifier: &RosettaTypes.OperationIdentifier{
+				Index: int64(count),
+			},
+			Type:   sdkTypes.FeeOpType,
+			Status: RosettaTypes.String(sdkTypes.SuccessStatus),
+			Account: &RosettaTypes.AccountIdentifier{
+				Address: address,
+			},
+			Amount: evmClient.Amount(amount, sdkTypes.Currency),
+		}
+		ops = append(ops, singleOp)
+
+		if shouldAdd {
+			doubleOp := &RosettaTypes.Operation{
+				OperationIdentifier: &RosettaTypes.OperationIdentifier{
+					Index: int64(count + 1),
+				},
+				RelatedOperations: []*RosettaTypes.OperationIdentifier{
+					{
+						Index: int64(count),
+					},
+				},
+				Type:   sdkTypes.FeeOpType,
+				Status: RosettaTypes.String(sdkTypes.SuccessStatus),
+				Account: &RosettaTypes.AccountIdentifier{
+					Address: transfer.To.String(),
+				},
+				Amount: evmClient.Amount(transfer.Value, sdkTypes.Currency),
+			}
+		ops = append(ops, doubleOp)
+		count++
+		}
+	}
+	return ops
+}
+
+func TransferOps(tx *evmClient.LoadedTransaction, startIndex int) []*RosettaTypes.Operation {
```

Comment:
Add a comment line for this. We should add a linter ideally to this repo which can detect this.